open-metadata-implementation/access-services/data-engine/data-engine-api/src/main/java/org/odpi/openmetadata/accessservices/dataengine/model/Process.java

```diff
@@ -32,14 +32,15 @@
     private List<PortImplementation> portImplementations;
     private List<PortAlias> portAliases;
     private List<LineageMapping> lineageMappings;
+    private UpdateSemantic updateSemantic;
 
     public Process() {
     }
 
     public Process(String qualifiedName, String name, String description, String latestChange,
                    List<String> zoneMembership, String displayName, String formula, String owner, OwnerType ownerType,
                    List<PortImplementation> portImplementations, List<PortAlias> portAliases,
-                   List<LineageMapping> lineageMappings) {
+                   List<LineageMapping> lineageMappings, UpdateSemantic updateSemantic) {
```

Comment:
In addition to this new constructor with the update semantic, should we have another constructor with a "default" update semantic (which simply passes through to this one)?  I'd suggest the default should be `REPLACE`, as it appears to be further below; but without an additional constructor that defaults this then any implementation using these methods would need to explicitly send `null` to this constructor (or it won't even compile) (?)